tests/pages/package-search.js

```diff
@@ -3,6 +3,28 @@ import { expect } from 'chai';
 import { convergeOn } from '../it-will';
 import { triggerChange } from '../helpers';
 
+function createPackageObject(element) {
```

Comment:
Should these be `createPackageListItemObject` or something to that effect to denote this object is for the list items on this page?